bosi/lib/constants.py

```diff
@@ -165,3 +168,18 @@
 UPGRADE_PYPI = ["ivs-debuginfo", "ivs", "horizon-bsn", "bsnstacklib"]
 UPGRADE_RPM = ["ivs-debuginfo", "ivs", "python-networking-bigswitch", "openstack-neutron-bigswitch-agent", "openstack-neutron-bigswitch-lldp", "python-horizon-bsn"]
 
+
+class MetaEnum(EnumMeta):
+    def __contains__(cls, item):
+        return item in cls.__members__
+
+
+class SriovBondMode(Enum):
+    """
+    This maps sriov bond mode to its corresponding MAC address to be used by
+    the LLDP script as system-desc.
+    """
+    __metaclass__ = MetaEnum
+
+    STATIC = '5c:16:c7:00:00:00'
```

Comment:
just confirmed with @padubidry over chat. he ran the changes with both static and lacp configured and its working as expected. merging the change.